labs/build-your-first-gadget/00 basic_tracer/template/program.bpf.c

```diff
@@ -37,7 +37,7 @@ static __always_inline int handle_open(struct syscall_trace_enter *ctx)
 
     // This is an Inspektor Gadget helper which checks if we should discard the
     // current data based on premade filters that we can set in Inspektor Gadget itself
-    // So if we to Inspektor Gadget that our gadget should only trace a container
+    // So if we set the Inspektor Gadget so that our gadget should only trace a container
     // named "nginx-40", this function will return 0 for all other containers
```

Comment:
I think this makes more sense IMO and fixes a logic bug in the comment too :D
```suggestion
    // So if we want that our gadget should only trace a container named "nginx-40",
    // this function will return 0 for the "nginx-40" container and 1 for all other containers
```

Reply:
This also needs to be updated in all other templates/solutions

Reply:
This needs to get updated into all other template and solution file